storm-core/src/clj/backtype/storm/config.clj

```diff
@@ -20,7 +20,8 @@
   (:import [backtype.storm Config ConfigValidation$FieldValidator])
   (:import [backtype.storm.utils Utils LocalState])
   (:import [org.apache.commons.io FileUtils])
-  (:require [clojure [string :as str]])
+  (:require [clojure.string :as str]
+            [backtype.storm.util :as util])
   (:use [backtype.storm log util]))
```

Comment:
Hadn't quite got to squashing all of the :imports, :requires and removing the :uses here yet